website/versioned_docs/version-0.67/integration-with-android-fragment.md

```diff
@@ -3,6 +3,8 @@ id: integration-with-android-fragment
 title: Integration with an Android Fragment
 ---
 
+import Tabs from '@theme/Tabs'; import TabItem from '@theme/TabItem'; import constants from '@site/core/TabsConstants';
```

Comment:
nit: can imports be multiline?

Reply:
I followed the same format used by others doc pages. can we include this as part of another PR where we make them all consistent?

Reply:
@ShikaSD @mdvacca IIRC in the past Docusuaurus throw an compile error when import were listed on the separate lines, but maybe it's not the case anymore. I agree, that it would be better to correct this formatting if possible, I will try to change this during next Docusuaurs bump, which will be soon. 🙂

Reply:
It looks like that this is still an issue, but the Docusuaurs team has found a workaround, which is wrapping the imports in `mdx-code-bock`:
* https://github.com/facebook/docusaurus/blame/main/website/versioned_docs/version-2.0.0-beta.17/guides/markdown-features/markdown-features-react.mdx#L10-L15